src/composables/useIsInLobby.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * @copyright Copyright (c) 2024, Dorra Jaouad <dorra.jaoued1@gmail.com>
+ *
+ * @author Dorra Jaouad <dorra.jaoued1@gmail.com>
+ * @author Daniel Calviño Sánchez <danxuliu@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { computed } from 'vue'
+
+import { useStore } from './useStore.js'
+import { PARTICIPANT, WEBINAR } from '../constants.js'
+
+/**
+ *
+ */
+export function useIsInLobby() {
+	const store = useStore()
+	const token = computed(() => store.getters.getToken())
+	const conversation = computed(() => store.getters.conversation(token.value))
+
+	const isModerator = computed(() => {
+		return conversation.value
+      && (conversation.value.participantType === PARTICIPANT.TYPE.OWNER
```

Comment:
```suggestion
		const type = conversation.value?.participantType
		return type === PARTICIPANT.TYPE.OWNER
```

to call computed props less
similar could be done for others

Reply:
> to call computed props less

`computed` are not "called". When it is calculated once, it is just a property with a value. It has almost no performance overhead (no more than using any reactive property in Vue).

Reply:
> to call computed props less

`computed` are not "called". When it is calculated once, it is just a property with a value. It has almost no performance overhead (no more than using any reactive property in Vue).